js/two_block.js

```diff
@@ -0,0 +1,118 @@
+//  1)
+const helloOleg = function(value) {
+
+    if(value === 'Mark') {
+        console.log(`Hi, ${value}`)
+    } else {
+        console.log(`Hello, ${value}`)
+    }
+}
+
+helloOleg('Pol');
+
+
+//  2)
+const hypotenuse = function(n, m) {
+
+    const result = (n*n) + (m*m);
+    console.log(Math.sqrt(result));
+}
+
+hypotenuse(4, 4);
+
+
+// 3)
+const min = function(a ,b) {
+    let minValue = 0;
+    let maxValue = 0;
```

Comment:
what the purpose of creating 2 variable ?

Reply:
Corrected